tests/e2e/e2e_test.go

```diff
@@ -184,6 +184,72 @@ var _ = Describe("Backstage Operator E2E", func() {
 			})
 		}
 	})
+
+	Context("Operator upgrade with existing instances", Label("upgrade"), func() {
+		const managerPodLabel = "control-plane=controller-manager"
+
+		// 0.1.3 is the version of the operator in the 1.1.x branch
+		var fromDeploymentManifest = filepath.Join(projectDir, "tests", "e2e", "testdata", "backstage-operator-0.1.3.yaml")
+		var (
+			crName = "bs1"
+			crPath = filepath.Join(projectDir, "examples", "bs1.yaml")
```

Comment:
Can we make it rely on some internal script and not one from "examples/bs1.yaml" for predictability?

Reply:
What do you mean by "some internal script"? To rewrite the whole test as a script?
We are supposed to maintain the example CRs, so it made sense to me for our E2E tests to leverage them.

Reply:
Nope, I meant yaml script, to use dedicated one, not from examples.

Reply:
Okay, I can do that, but still don't understand the benefit. In the scope of end-to-end tests, I want to make sure the examples are maintained and work, and having one of them used for testing the upgrade path here seems to make sense to me, doesn't it?

Reply:
Well, to me tests and example it is 2 different things :) 
Your point makes sense too...
OK, it is a different topic, let's go as it is.